lib/rails/generators/intercom/config/intercom.rb.erb

```diff
@@ -21,8 +21,7 @@ IntercomRails.config do |config|
   <%- end -%>
 
   # == Intercom API Key
-  # This is required for some Intercom rake tasks like importing your users;
-  # you can generate one at https://app.intercom.io/apps/api_keys.
+  # You can generate one at https://app.intercom.io/apps/api_keys.
```

Comment:
I don't think we use REST API keys for anything other than the import task. We should get rid of this and all the underlying config unless there is another use for it that I'm not seeing.